tests/kuttl/test-suites/default/watcher-api/03-deploy-watcher-api.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   name: watcherapi-kuttl
 spec:
   databaseInstance: "openstack"
+  secret: "osp-secret"
```

Comment:
Given that the idea is that Watcher will create a secret with the minimum set of information required for the watcherapi, watcherdecisionengine and watcherapplier, we should use a different secret for watcherapi that the top level osp-secret this in kuttl tests containing only the required parameters instead of the entire osp-secret.

Also, in future, watcherapi may need some parameters which are not included in osp-secret.

Reply:
do you think it would be better to create some temporary secret in the kuttl tests and use that? By the time we integrate all the CRs into watcher we'll need to modify all these tests anyway

Reply:
Yes, that was my idea.  That would also help us when creating the integration between first level watcher and sub-CRs to identify what information should be included in the secret provided from Watcher to WatcherAPIs and the rest.

Reply:
ack, makes sense to be explicit, let me update the PR with the change

Reply:
done, kuttl tests are passing locally, let's see if CI is green as well